### PR TITLE
Moved contacts table to different component

### PR DESCRIPTION
**Pull request by shreyakurian02**
Moved Contacts table to different component

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                </tr>
+              </React.Fragment>
+            ))}
+        </tbody>
+      </table>
+    </>
+  );
+};
+
+export default Table;
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:123]: Please rename the file and component to `ContactList.jsx`.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
-import { Search, BurgerMenu, MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Search, BurgerMenu } from "@bigbinary/neeto-icons";
 import { PageLoader } from "neetoui";
-import { Button, Pagination, Dropdown, Checkbox, Avatar } from "neetoui/v2";
-import { Input, Typography } from "neetoui/v2";
+import { Button, Pagination } from "neetoui/v2";
+import { Input } from "neetoui/v2";
 import { Header } from "neetoui/v2/layouts";
 import { Container } from "neetoui/v2/layouts";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:9]: Please remove duplication for import statements.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 3]
```diff
@@ -1,16 +1,17 @@
 import React, { useState, useEffect } from "react";
 
-import { Search, BurgerMenu, MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Search, BurgerMenu } from "@bigbinary/neeto-icons";
 import { PageLoader } from "neetoui";
-import { Button, Pagination, Dropdown, Checkbox, Avatar } from "neetoui/v2";
-import { Input, Typography } from "neetoui/v2";
+import { Button, Pagination } from "neetoui/v2";
+import { Input } from "neetoui/v2";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:6]: Please remove duplication for import statements.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 4]
```diff
@@ -1,16 +1,17 @@
 import React, { useState, useEffect } from "react";
 
-import { Search, BurgerMenu, MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Search, BurgerMenu } from "@bigbinary/neeto-icons";
 import { PageLoader } from "neetoui";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:4]: Is this fixed in the other PR?

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+import { MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Dropdown, Checkbox, Avatar } from "neetoui/v2";
+import { Typography } from "neetoui/v2";
+
+const Table = ({ setShowDeleteAlert }) => {
+  return (
+    <>
+      <table
+        className={`neeto-ui-table neeto-ui-table--checkbox neeto-ui-table--actions`}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:11]: We don't need `{}` here.

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 6]
```diff
@@ -0,0 +1,123 @@
+import React from "react";
+
+import { MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Dropdown, Checkbox, Avatar } from "neetoui/v2";
+import { Typography } from "neetoui/v2";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:5]: Please remove this duplication.

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+                    </div>
+                  </td>
+                </tr>
+              </React.Fragment>
+            ))}
+        </tbody>
+      </table>
+    </>
+  );
+};
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:121]: Please split this into `Header and Body`.

**Review by karthiknmenon — COMMENTED** [review]
@shreyakurian02 Please check the suggestions.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx (hunk 4 above)
Comment (shreyakurian02) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx:4]: yes

**Pull request merged**